sotopia/cli/benchmark/benchmark.py

```diff
@@ -262,9 +263,57 @@ def _set_up_logs(
     )
 
 
+def save_to_jsonl(
+    model_rewards_dict: Dict[str, Dict[str, float]],
+    partner_model: str,
+) -> None:
+    simplified_model_name = partner_model.split("/")[-1]
+    output_fn = f"./models_vs_{simplified_model_name}.jsonl"
+    outputs: List[str] = []
+    for model, rewards in model_rewards_dict.items():
+        formatted_reward = OrderedDict(
+            {
+                "model_name": model,
+                **{
+                    f"{v[0]} {v[1]}": rewards[k]
+                    for k, v in dimension_range_mapping.items()
+                },
+            }
+        )
+        outputs.append(json.dumps(formatted_reward))
+    with open(output_fn, "w") as f:
+        f.write("\n".join(outputs))
+
+    print(f"Output saved to {output_fn}")
+
+
+default_model_list: List[str] = [
```

Comment:
Default model list could just be a list of one model?

Reply:
Okay then we go ahead with gpt-4o